addons/account_fleet/models/fleet_vehicle_log_services.py

```diff
@@ -24,8 +28,7 @@ def _inverse_amount(self):
     @api.depends('account_move_line_id.price_subtotal')
     def _compute_amount(self):
         for log_service in self:
-            if log_service.account_move_line_id:
```

Comment:
It isn't since the compute would only be triggered once the price subtotal of the account move line is edited;
- If there is no account move line, it will never be updated;
- If there is an account move line, it will and should be updated;
- The only way to remove the account move line is by deleting the record, which will automatically remove the log service alongside with it
- It isn't possible to add an account move to the log service fonctionnaly, but even if we could we should update the value in this case as well
I believe that's every case related to that compute, and in any case removing the extra check doesn't seem to cause any issue :thinking: If you feel like there might be one I can add it back :)

Reply:
no indeed, there is `account_move_line_id.price_subtotal` in trigger